index.js

```diff
@@ -1,9 +1,12 @@
 "use strict";
 
 const codes = require("./codes.json");
+const supportedLocales = require("./supportedLocales.json");
 const removeDiacritics = require("diacritics").remove;
 const registeredLocales = {};
 
+exports.supportedLocales = supportedLocales;
```

Comment:
Can we remove this and use getSupportedLanguages() instead? This will make it easier to change the datastructure later if needed?